listenbrainz_spark/stats/sitewide/entity.py

```diff
@@ -31,6 +31,18 @@
 }
 
 
+def get_listen_count_limit(stats_range: str) -> int:
```

Comment:
For now, the limit is set to 500 for all stats. However, we probably will want different listen count limits for different time ranges. Should we also make the limit change depending on whether its a artist stat or recording stat or release stat?

Reply:
I think for now that is ok -- if things feel like they are wrong, we'll go look again.